events/views.py

```diff
@@ -45,3 +56,164 @@ def update_event(request, pk):
+            is_meeting_formset_invalid = check_dates_constraint(form, event_instance, request)
+            if is_meeting_formset_invalid:
+                break
+        if is_meeting_formset_invalid:
+            event_instance.delete()
+            is_valid_formsets[0] = False
+        else:
+            saving_all_optional_meeting_dates(event_creator, event_instance, optional_meetings_formset)
+    else:
+        event_instance.delete()
+        is_valid_formsets[0] = False
+
+
+def check_participant_formset(request, event_instance, meeting_participants_formset, is_valid_formsets):
+    if meeting_participants_formset.is_valid():
+        for form in meeting_participants_formset:
+            if form.is_valid():
+                try:
+                    participant_email = form.cleaned_data.get('participant_email')
+                    if participant_email:
+                        user_instance = User.objects.get(email=participant_email)
+                        participant = EventParticipant(event_id=event_instance, user_id=user_instance, is_creator=False)
+                        participant.save()
+                except User.DoesNotExist:
+                    messages.warning(request, f"There is not user with the email: {participant_email}")
+                    event_instance.delete()
+                    is_valid_formsets[0] = False
+                except ValidationError:  # duplication of the same participant email
+                    pass
+    else:
+        event_instance.delete()
+        is_valid_formsets[0] = False
+
+
+@login_required(login_url=LOGIN_PAGE)
+def create_meeting(request):
+
+    OptionalMeetingDateFormSet = formset_factory(OptionalMeetingDateForm,
+                                                 formset=BaseOptionalMeetingDateFormSet,
+                                                 max_num=10, extra=0)
+    MeetingParticipantsFormset = formset_factory(ParticipantForm,
+                                                 formset=BaseParticipantFormSet,
+                                                 max_num=10, extra=0)
+    if request.method == 'POST':
+        create_event_form = EventCreationForm(request.POST, user_id=request.user)
+        optional_meetings_formset = OptionalMeetingDateFormSet(request.POST, prefix='optional_meetings')
+        meeting_participants_formset = MeetingParticipantsFormset(request.POST,
+                                                                  prefix='participants', user_id=request.user)
+
+        is_valid_formsets = [True]
+        if create_event_form.is_valid():
+            event_instance = create_event_form.save()
+            event_creator = EventParticipant.objects.get(event_id=event_instance, user_id=request.user, is_creator=True)
+
+            check_optional_meeting_dates_formset(request, event_instance,
+                                                 event_creator, optional_meetings_formset, is_valid_formsets)
+            if is_valid_formsets[0] is True:
```

Comment:
```suggestion
            if is_valid_formsets[0]:
```

Please remove the is True

Reply:
Done 👍